<div id="header" align="center">
  <div id="header" align="center">
    <picture>
          <img src="https://media.giphy.com/media/3ornk57KwDXf81rjWM/giphy.gif" width="500"/>
    </picture>
  </div>
    <div id="badges">
      <a href="https://www.linkedin.com/in/james-boot-phd-051002a0/">
        <img src="https://img.shields.io/badge/LinkedIn-blue?logo=linkedin&logoColor=white&style=for-the-badge" alt="LinkedIn Badge"/>
    </div>
  <picture>
    <img src="https://komarev.com/ghpvc/?username=jamesboot&style=flat-square&color=blue" alt=""/>
  </picture>
</div>

# :mega: About me

## :alarm_clock: TL;DR
- ðŸ”­ Iâ€™m currently working on: implementing ML models in analysis & pipeline automation & spatial transcriptomics neighbourhood analysis (interactVis)!
- ðŸŒ± Iâ€™m currently learning: NextFlow!
- ðŸ’¬ Ask me about: Anything! I'll try and help as best as I can!
- ðŸ“« How to reach me: LinkedIn
- âš¡ Fun fact: I went to high school in China! And my profile picture is AI generated (surprise!).

## :pencil: Key publications
- https://doi.org/10.7554/eLife.77335
- https://doi.org/10.1038/s41467-021-26297-6

## :page_facing_up: Extended Intro
I am a Senior Bioinformatician at The Francis Crick Institute in London, I work within the Bioinformatics and Biostatistics science technology platform providing analysis support to research scientists. I learnt to code (R) and the basics of bioinformatics during my PhD, which was focussed on the epigenetic and transcriptomic profiles of Glioblastoma brain tumours. Following my PhD I took up a position as Bioinformatician within the Genome Centre at QMUL for 3 years, before moving to The Crick. Being predominantly self-taught; I love the continual learning process of coding and bioinformatics which makes everyday exciting! 

I have experience with many dataset types including: 
- Bulk-RNAseq
- Single-cell-RNAseq
- Spatial transcriptomics
- ChIPseq
- RRBS
- Methylation arrays
- WGS
- Flow cytometry

My services range from support to full analysis pipelines for clients, which may include where appropriate: 
- Quality control
- Filtering
- Normalisation
- Dimensionality reduction
- Clustering
- Exploratory analysis
- Batch correction
- Differential analysis
- Pathway analysis and more.


